packages/kcms/src/team/service/entry.ts

```diff
@@ -1,63 +1,51 @@
-import { EntryRepository } from '../models/repository.js';
-import { Team, TeamCreateArgs } from '../models/team.js';
+import { TeamRepository } from '../models/repository';
+import { Team, TeamID } from '../models/team';
 import { Option, Result } from '@mikuroxina/mini-fn';
-import { SnowflakeIDGenerator } from '../../id/main.js';
 
+/**
+ * 当日参加を行うService
+ */
 export class EntryService {
-  private readonly repository: EntryRepository;
-  private readonly idGenerator: SnowflakeIDGenerator;
-  constructor(repository: EntryRepository, idGenerator: SnowflakeIDGenerator) {
-    this.repository = repository;
-    this.idGenerator = idGenerator;
-  }
-
-  async create(input: Omit<TeamCreateArgs, 'id'>): Promise<Result.Result<Error, Team>> {
-    const id = this.idGenerator.generate<Team>();
-    if (Result.isErr(id)) {
-      return Result.err(id[1]);
+  constructor(private readonly teamRepository: TeamRepository) {}
+
+  /**
+   * チームの出欠を登録します
+   * @param teamID 出席するチームのID
+   * @returns 出席登録後のチームデータ
+   */
+  async entry(teamID: TeamID): Promise<Result.Result<Error, Team>> {
```

Comment:
```suggestion
  async enter(teamID: TeamID): Promise<Result.Result<Error, Team>> {
```
もしくは
```suggestion
  async makeEntry(teamID: TeamID): Promise<Result.Result<Error, Team>> {
```
とか？(`cancelEntry`との対比)